README.md

```diff
@@ -38,9 +43,10 @@ The plugin can be used in Jenkins as a pipeline task or as a build step.
 
 ### 2. Configuration
 Make sure that the Path is updated to point to the corresponding executables, like `mvn`, `pip` etc.
-<h5>Customization</h5>
-<p>
-There is a way to set a custom Maven path using <em>Environment Variables</em>. 
+
+#### Customization
+
+There is a way to set a custom Package Managers paths using <em>Environment Variables</em>. 
```

Comment:
Simplify the sentence. Is there a need to emphasize "Environment Variables"?  I would only emphasize if you're calling out an actual UI element.  Here is my suggestion:

`There is a way to set a custom Package Managers paths using <em>Environment Variables</em>. ` -->
`To set a custom path for package managers use environment variables.`

Reply:
Done, Thanks.